classic/bootstrap.sh

```diff
@@ -2,19 +2,59 @@
 
 function bootstrap-azure {
     echo "Bootstrap Azure resources for ARO install with RP"
+    set -e
+
+    if [[ -z "${AZURE_SUBSCRIPTION_ID}" ]]; then
+        echo ">> AZURE_SUBSCRIPTION_ID is not set"
+        exit 1
+    fi
+
+    if [[ -z "${AZURE_CLUSTER_RESOURCE_GROUP}" ]]; then
+        echo ">> AZURE_CLUSTER_RESOURCE_GROUP is not set"
+        exit 1
+    fi
+
+    echo "Creating resource group ${AZURE_CLUSTER_RESOURCE_GROUP}"
+    az group create \
+        --subscription ${AZURE_SUBSCRIPTION_ID} \
+        --resource-group ${AZURE_CLUSTER_RESOURCE_GROUP} \
+        --location eastus
```

Comment:
Given the known capacity issues here until possibly November, should we choose a different location?